dogen/templates/template.jinja

```diff
@@ -102,6 +102,7 @@ RUN [ "bash", "-x", "/tmp/scripts/{{ script.package }}/{{ script.exec }}" ]
 {% if cct %}
 ADD cct /tmp/cct/
 ENV CCT_MODULES_PATH=/tmp/cct/
+USER {{ cct.user }}
```

Comment:
Yah, that's a different topic. Ticket?

You are right, after looking at the template again there is no need to specify back the user since it's done later either case.